components/nimbus/src/enrollment.rs

```diff
@@ -521,12 +523,22 @@ impl EnrollmentStatus {
                 ref mut enrollment_id,
                 ..
             } => *enrollment_id = Uuid::nil(),
-            EnrollmentStatus::NotEnrolled { .. } | EnrollmentStatus::Error { .. } => (),
+            EnrollmentStatus::NotEnrolled { .. } | EnrollmentStatus::EnrollError { .. } => (),
         };
         updated
     }
 }
 
+pub(crate) trait SharedExperimentMetadata {
```

Comment:
Nit: why is this called `Shared`?

Reply:
I called it `Shared` because the information the trait's methods returns is shared across a number of different types — happy to just call it `ExperimentMetadata` if that makes more sense. Or something else entirely